airflow/api_fastapi/core_api/routes/public/dag_run.py

```diff
@@ -383,9 +417,16 @@ def trigger_dag_run(
         raise HTTPException(status.HTTP_400_BAD_REQUEST, str(e))
 
 
-@dag_run_router.post("/list", responses=create_openapi_http_exception_doc([status.HTTP_404_NOT_FOUND]))
+@dag_run_router.post(
```

Comment:
```suggestion
@dag_run_router.get(
```

should it be get instead?

Reply:
No. This is a post-call. Data will be passed in the body instead of the URL. Its as per the legacy implementation

Reply:
https://airflow.apache.org/docs/apache-airflow/stable/stable-rest-api-ref.html#operation/get_dag_runs_batch

Reply:
i think, the reason for this is that the data isn't passed as query params. rather, in body

Reply:
ah, got it. Thanks!